src/wmtk/operations/tri_mesh/VertexTangentialSmooth.cpp

```diff
@@ -32,8 +30,7 @@ bool VertexTangentialSmooth::before() const
 bool VertexTangentialSmooth::execute()
 {
     const Eigen::Vector3d p = m_pos_accessor.vector_attribute(input_tuple());
-    OperationSettings<tri_mesh::VertexSmooth> op_settings;
-    tri_mesh::VertexSmooth smooth_op(mesh(), input_tuple(), m_settings.smooth_settings);
+    tri_mesh::VertexLaplacianSmooth smooth_op(mesh(), input_tuple(), m_settings.smooth_settings);
```

Comment:
This operation is just the parent class's smooth right? just invoke the parent class's execute rather than creating another instasnce of the parent class (you have two VertexLaplacianSmooth in existence here)